client/my-sites/checkout/controller.jsx

```diff
@@ -62,9 +62,10 @@ export function checkout( context, next ) {
 	const jetpackPurchaseNonce = context.query.purchaseNonce;
 	const isUserComingFromLoginForm = context.query?.flow === 'logged-out-checkout';
 	const isJetpackCheckout =
-		context.pathname.includes( '/checkout/jetpack' ) &&
-		( isLoggedOut || isUserComingFromLoginForm ) &&
-		( !! jetpackPurchaseToken || !! jetpackPurchaseNonce );
+		( context.pathname.includes( '/checkout/jetpack' ) &&
+			( isLoggedOut || isUserComingFromLoginForm ) &&
+			( !! jetpackPurchaseToken || !! jetpackPurchaseNonce ) ) ??
```

Comment:
I'm sure this was added for a reason, but I'm curious about the logic here; the nullish coalescing operator evaluates to its right hand side value when the left hand site is null or undefined, but I think that all the left hand side conditions here always evaluate to a boolean. Is there a way that this `?? false` could be triggered?

Reply:
Oh you're right. No, I don't think the `?? false` could ever be triggered. It can be removed.
At one point I added this in attempt to fix a TS linting warning but then later found it was not needed, but forgot to remove it. 🤦‍♂️  Good catch, thank you!